File: Lab 6/t3.go
package main

import (
  "fmt"
  "os"
  "bufio"
  "strings"
)

func main() {
  os.Exit(t3_main())
}

func t3_main() int {
  buf, e := bufio.NewReader(os.Stdin).ReadString('\n')
  if e != nil {
    fmt.Println("Read Error:", e)
    return 1
  }

  fmt.Println(strings.Count(buf, "a"))
  return 0
}

File: Lab 6/t3.rs
fn main() {
    let mut buf = String::new();
    match std::io::stdin().read_line(&mut buf) {
        Ok(_) => {},
        Err(e) => {
            println!("Error while reading: {}", e);
            std::process::exit(1);
        }
    };

    let mut counter:i32 = 0;
    for i in buf.chars() {
        if i == 'a' {
            counter += 1;
        }
    }

    println!("{}", counter);
}
